pkg/hypervisor/hypervisor.go

```diff
@@ -168,10 +176,16 @@ func (m *Hypervisor) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 			r.Delete("/visors/{pk}/routes/{rid}", m.deleteRoute())
 			r.Get("/visors/{pk}/loops", m.getLoops())
 			r.Get("/visors/{pk}/restart", m.restart())
-			r.Post("/visors/{pk}/exec", m.exec())
 		})
 	})
 
+	r.Route("/pty", func(r chi.Router) {
```

Comment:
Why isn't it a part of the `/api`?

Reply:
@nkryuchkov Because it serves a WebUI so I thought it shouldn't be part of `/api`. What do you think?

Reply:
@evanlinjin Yeah, I agree.